### ISSUE TITLE
Add Contribution.md file

### ISSUE THREAD

**Issue by GowthamGoush** [GowthamGoush/Amazing_Sites#1]
Participant is requested to add contribution.md file

**Comment by Rangaraj-baskar** [comment]
Looks interesting ! Would love to work on this !

**Comment by stutimongia2024** [comment]
I would like to contribute

**Comment by GowthamGoush** [comment]
Hey, @stutimongia2024 currently @Rangaraj-baskar is working on this issue. Nevertheless feel free to create a new issue and add any website that you wish to see in the list.

**Comment by abhikalparya** [comment]
Hey @GowthamGoush, please assign me this issue. I will add several amazing websites in this readme file.

**Comment by Rangaraj-baskar** [comment]
@abhikalparya We encourage your enthusiasm ! But I am currently working on this. Feel free to work on other parts.

**Comment by GowthamGoush** [comment]
@abhikalparya, feel free to create a new issue for the website which you wish to add. Good luck!

**Comment by GowthamGoush** [comment]
@stutimongia2024 and @abhikalparya any of you would like to work on this?

**Comment by abhikalparya** [comment]
Yes @GowthamGoush, please assign me this issue. I will create a Contribution.md file.
